boltdesignsystem/bolt
# DS-419: Fix support for Edge 18/44

## Jira

https://pegadigitalit.atlassian.net/browse/DS-419

## Summary

Fix JS error in Edge 18 that says `Expected '('`.

## Details

I believe this error is thrown because Edge 18 doesn't know how to handle [`...rest` syntax](https://stackoverflow.com/questions/53628191/edge-script1028-expected-identifier-string-or-number). I assume this feature was automatically transpiled before we upgraded Babel, as it does not happen prior to Bolt `3.2.0`.

Explicitly adding `Edge 18` to our browserslist-config fixes the bug in Edge 18 on Browserstack.

I'm also adding `Edge 18` to our Pattern Lab Babel config to fix a JS error on the outer PL page (as opposed to the iframe). You may still notice a JS error on the outer PL page, but at least the page works. I'm not going to go any further to debug PL in Edge 18.

## How to test

- Reviewed deployed site in Edge 18 in Browserstack.
  - [Page](https://hotfix-ds-419-edge-support.boltdesignsystem.com/pattern-lab/?p=components-accordion) in PL
  - [Page](https://hotfix-ds-419-edge-support.boltdesignsystem.com/pattern-lab/patterns/40-components-accordion-05-accordion/40-components-accordion-05-accordion.html) in new tab
- Look at a few demos and verify there is no `Expected '('` error or any other major JS errors.
- @colbytcook please take a look on your real Windows machine with Edge 42/44 if possible to verify the same.
